MusicApp.SongService.Application/DTOs/SongInputDto.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace MusicApp.SongService.Application.DTOs;
+
+public class SongInputDto
```

Comment:
In which way will you connect song with artist?

Reply:
I will change the logic of adding an artist to the database in message broker pull request